cmd/collector/main.go

```diff
@@ -150,6 +170,14 @@ func getDefaultLoggingConfigPath() string {
 	return logging.DefaultConfigPath
 }
 
+func getDefaultResolverConfigPath() string {
```

Comment:
Unlike the other configs (logging, config, etc) there is no default resolver config. The default behavior will be to skip configuring a custom dns resolver.